albumentations/augmentations/domain_adaptation.py

```diff
@@ -75,6 +83,18 @@ def apply_histogram(img, reference_image, blend_ratio):
     return img
 
 
+@preserve_shape
+def adapt_pixel_distribution(
+    img: np.ndarray, ref: np.ndarray, transform_type: str = "pca", weight: float = 0.5
+) -> np.ndarray:
+    initial_type = img.dtype
+    transformer = {"pca": PCA, "standard": StandardScaler, "minmax": MinMaxScaler}[transform_type]()
+    adapter = DomainAdapter(transformer=transformer, ref_img=ref)
+    result = adapter(img).astype("float32")
+    blended = (img.astype("float32") * (1 - weight) + result * weight).astype(initial_type)
```

Comment:
`cv2.addWeighted` might be faster there
https://docs.opencv.org/4.5.2/d2/de8/group__core__array.html#gafafb2513349db3bcff51f54ee5592a19

Reply:
For 100*100 images the difference is 7 µs (27 vs 34) on my laptop - I'd stick with `numpy` version for its better readability.